src/referencePackages/src/nuget.common/6.2.2/NuGet.Common.6.2.2.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
```

Comment:
Can you regenerate these again off of the tip tooling.  I am not expecting and net4* tfms to be included by default anymore after a change I submitted yesterday.  If you simply regenerate, you will have to manually remove the net472 source files that were previously generated.

Reply:
I still see all of the net472 artifacts.  You will need to manually delete the net472 artifacts that were previously generated.

Reply:
Yeah, I simply forgot to `git add` the removal. My bad, should be fixed now